rpc/api.proto

```diff
@@ -59,6 +59,11 @@ service WalletLoaderService {
 	rpc FetchHeaders(FetchHeadersRequest) returns (FetchHeadersResponse);
 }
 
+service TicketBuyerService {
+    rpc StartTicketPurchase (StartTicketPurchaseRequest) returns (StartTicketPurchaseResponse);
+    rpc StopTicketPurchase (StopTicketPurchaseRequest) returns (StopTicketPurchaseResponse);
```

Comment:
I think it would be clearer if we renamed these to "StartBuyer" or "StartAutoBuyer"